app/views/layouts/_header.html.haml

```diff
@@ -1,16 +1,26 @@
-%header.main-header
-  %nav.secondary-navigation
-    - unless write_access?
+%nav.main-navigation{ class: [write_access? && "has-write-access"] }
+  .main-header-wrap
+    %header.main-header
+      %h1.site-title
```

Comment:
I believe this should only be the `h1` on the home page and be a `div` everywhere else.